tools/jdbc/src/test/java/org/duckdb/test/TestDuckDBJDBC.java

```diff
@@ -329,6 +329,54 @@ public static void test_enum() throws Exception {
 		conn.close();
 	}
 
+	public static void test_timestamp_ms() throws Exception {
+		String expectedString = "2022-08-17 12:11:10.999";
+		String sql = "SELECT '2022-08-17T12:11:10.999'::TIMESTAMP_MS as ts_ms";
+		assert_timestamp_match(sql, expectedString, "TIMESTAMP_MS");
+	}
+
+	public static void test_timestamp_ns() throws Exception {
+		String expectedString = "2022-08-17 12:11:10.999999";
```

Comment:
This appears to truncate to the microsecond value. 

Could be a bug with how I've implemented it, but it apperas as though this might also be how data is returned for NS in the CLI: 

```
D SELECT '2022-08-17T12:11:10.999999999'::TIMESTAMP_NS as ts_ns;
┌────────────────────────────┐
│           ts_ns            │
├────────────────────────────┤
│ 2022-08-17 12:11:10.999999 │
└────────────────────────────┘
```

is this expected?

Reply:
Yes, that is fine. We don't have specialized code for converting `timestamp_ns` to varchar so it uses the default timestamp conversion code instead.